lib/ansible/modules/cloud/azure/azure_rm_resourcegroup.py

```diff
@@ -162,7 +162,7 @@ def exec_module(self, **kwargs):
                 if update_tags:
                     changed = True
 
```

Comment:
i was wondering, perhaps we could normalize location before, e.g.:

self.location = normalize_location_name(self.location)

but just a suggestion. not sure perhaps that would have some disadvantages...